### PR TITLE
chore: make 2.0 RC builds optional

### PR DESCRIPTION
**Pull request by gshif**
Closes # https://github.com/influxdata/influxdb/issues/19892
- Update regex to be able to tag GA releases
- Remove selenium acceptance test run from nightly build

Describe your proposed changes here.

- [ ] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [ ] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [ ] Rebased/mergeable
- [ ] Tests pass
- [ ] http/swagger.yml updated (if modified Go structs or API)
- [ ] Feature flagged (if modified API)
- [ ] Documentation updated or issue created (provide link to issue/pr)
- [ ] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -558,31 +558,31 @@ workflows:
...
             tags:
-              only: /^v[0-9]+\.[0-9]+\.[0-9]+-(rc|alpha|beta)\.[0-9]+$/
+              only: /^v[0-9]+\.[0-9]+\.[0-9]+(-(rc|alpha|beta)\.[0-9]+)$/
       - golint:
           filters:
             branches:
               ignore: /.*/
             tags:
-              only: /^v[0-9]+\.[0-9]+\.[0-9]+-(rc|alpha|beta)\.[0-9]+$/
+              only: /^v[0-9]+\.[0-9]+\.[0-9]+(-(rc|alpha|beta)\.[0-9]+)$/
```
Comment (russorat) [inline .circleci/config.yml:567]: i think you need a `?` after the last parenthesis to make it optional

Comment (gshif) [reply .circleci/config.yml]: You are correct, made a change.

File: .circleci/config.yml [hunk 2]
```diff
@@ -558,31 +558,31 @@ workflows:
             branches:
               ignore: /.*/
             tags:
-              only: /^v[0-9]+\.[0-9]+\.[0-9]+-(rc|alpha|beta)\.[0-9]+$/
+              only: /^v[0-9]+\.[0-9]+\.[0-9]+(-(rc|alpha|beta)\.[0-9]+)?/
```
Comment (russorat) [inline .circleci/config.yml:561]:
```suggestion
              only: /^v[0-9]+\.[0-9]+\.[0-9]+(-(rc|alpha|beta)\.[0-9]+)?$/
```

Comment (gshif) [reply .circleci/config.yml:561]: @russorat you are suggesting to keep a `$` at the end?

Comment (gshif) [reply .circleci/config.yml]: Added `$` sign at the end of the expression

**Review by russorat — APPROVED** [review]
LGTM

**Pull request merged**